libraries/AP_AHRS/AP_AHRS_NavEKF.cpp

```diff
@@ -1420,15 +1451,17 @@ uint32_t AP_AHRS_NavEKF::getLastYawResetAngle(float &yawAng) const
 {
     switch (ekf_type()) {
 
-    case 2:
-    default:
+    case EKFType::NONE:
+        return 0;
```

Comment:
This is a behavioural change.  I think it is warranted as using EKF2's state is kind of random (we may actually be using EKF3 for everything else!)